src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java

```diff
@@ -206,6 +232,19 @@ protected boolean isConsideredNull(Object propertyValue) {
         return values.entrySet().stream().anyMatch(entry -> Strings.isEmpty(entry.getValue()));
     }
 
+    @Override
+    public void parseValues(Object entity, Values values) {
+        MultiLanguageString multiLanguageString = getMultiLanguageString(entity);
+        Map<String, String> mlsMap = new HashMap<>(multiLanguageString.data());
+        mlsMap.put(MultiLanguageString.FALLBACK_KEY, values.at(0).getString());
+        setValue(entity, mlsMap);
+    }
+
+    @Override
+    @SuppressWarnings("unchecked")
+    protected void checkNullability(Object propertyValue) {
```

Comment:
// Intentionally left empty as logic is implemented in onBeforeSaveChecks

-> Sonarlint should normally mark this with an inspection (leaving method empty...